Posts in events

nilsen340 opened the pull request:
Given the fact that we provide posts with the event endpoint we should map them into the events returned by the SDK. 

@onurakpolat @xla could you please do a quick review?

onurakpolat commented on Classes/TGEventManager.m:
```diff
@@ -504,11 +504,20 @@ - (void)handleSingleEventResponse:(NSDictionary*)jsonResponse withError:(NSError
 
     for (NSDictionary *eventData in eventDictionaries) {
         TGEvent *newEvent = [TGEvent createOrLoadWithDictionary:eventData];
+        if([newEvent.type  isEqual: @"tg_like"]) {
```
We should start using constants more often for things like `events` or `tg__like` but ok for now.

nilsen340 replied:
actually, yes. Ill add that.

onurakpolat commented:
@nilsen340 please more context into description next time ;)

Other than that :+1:

nilsen340 commented:
@onurakpolat I've fixed what we discussed earlier and also added user parsing on the event fetch. This is what made events sometimes miss users, as if events were fetched before users there would be no user to add to the event object. Feel free to have another look at it.

onurakpolat commented:
:+1:
